streams/src/test/java/org/apache/kafka/streams/integration/GlobalKTableIntegrationTest.java

```diff
@@ -291,6 +302,25 @@ private void produceTopicValues(final String topic) throws Exception {
                 mockTime);
     }
 
+    private void produceAbortedMessages() throws ExecutionException, InterruptedException {
```

Comment:
nit: in tests, it better to just declare `throws Exception` -- it does not provide value to spell out the different types in test and introduces "noise"

I realized, that the existing code also does declare different types of exception. Would you mind to clean this up, too?